go/luhn/cases_test.go

```diff
@@ -0,0 +1,97 @@
+package luhn
```

Comment:
case_test.go is needed unless you have case.go file. 
you can move test cases to luhn_test.go